pkg/apiserver/model/application.go

```diff
@@ -253,3 +266,29 @@ func (a *ApplicationRevision) Index() map[string]string {
 	}
 	return index
 }
+
+// ApplicationWebhook is the model for webhook request
+type ApplicationWebhook struct {
+	Model
+	AppPrimaryKey string `json:"appPrimaryKey"`
+	Token         string `json:"token"`
```

Comment:
Including workflow name? The user doesn't care about the app name, so doesn't care about the workflow name.

Reply:
It‘s a optional param, default to default workfkow. We need to choose workflow when we’re depoying.